MTM-Backend/prisma/schema.prisma

```diff
@@ -2,20 +2,70 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
+    provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "mysql"
-  url      = env("DATABASE_URL")
+    provider = "mysql"
+    url      = env("DATABASE_URL")
```

Comment:
code formats

Reply:
Prettier isn't changing the current formatting for schema.prisma. I sent an image on slack.